linker.ld

```diff
@@ -1,5 +1,8 @@
 ENTRY(__start)
 
+gNumMusicPlayers = 0x20;
```

Comment:
Why is this located here?

Reply:
I got this from pret.
I tried to get it to match a different way, but it didn't work.
I think it has something to do with these being variables inside th m4a lib that are set by the game.